geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/utils/StorageJobUtils.scala

```diff
@@ -41,11 +44,12 @@ object StorageJobUtils extends LazyLogging {
 
   // hadoop 3 API
   private def distCpOptions3(src: Path, dest: Path): DistCpOptions = {
-    val clas = Class.forName("org.apache.hadoop.tools.DistCpOptions.Builder")
+    val clas = Class.forName("org.apache.hadoop.tools.DistCpOptions$Builder")
```

Comment:
Ok, should we document that the bulk load job requires Hadoop 3?  Or is that obvious?  (Maybe Accumulo 2 requires Hadoop 3?)

Reply:
Also, relative to this, is the distcp used on the HBase bulk import job?  E.g. does the Hadoop 2/3 difference matter for that?

Reply:
this should work with both hadoop 2 and 3 (although i didn't test it), there's a different method that gets called for hadoop 2. i think this hadoop 3 code has been broken all along and we just never noticed.

this is not used for the hbase bulk load.

Reply:
i did document that it's only implemented for accumulo 2, although i'm sure we could do accumulo 1 if we wanted but there didn't seem to be a pressing reason.

Reply:
Makes sense.